docs/guides/create-app/Component.md

```diff
@@ -8,10 +8,15 @@ Previously, we saw that App requires a Component for rendering. We will write ou
 import { createComponent } from 'frint';
 
 export default createComponent({
+  // required method
```

Comment:
may be update the `createComponent.js` file too, and throw an `Error` if `render` is not passed as an option.